src/OrchardCore.Modules/OrchardCore.Media/Assets/js/MediaApp/folderComponent.js

```diff
@@ -162,27 +162,27 @@ Vue.component('folder', {
                 return;
             }
 
-            if (!confirm($('#moveMediaMessage').val())) {
-                return;
-            }            
-
-            $.ajax({
-                url: $('#moveMediaListUrl').val(),
-                method: 'POST',
-                data: {
-                    __RequestVerificationToken: $("input[name='__RequestVerificationToken']").val(),
-                    mediaNames: mediaNames,
-                    sourceFolder: sourceFolder,
-                    targetFolder: targetFolder
-                },
-                success: function () {
-                    bus.$emit('mediaListMoved'); // MediaApp will listen to this, and then it will reload page so the moved medias won't be there anymore
-                },
-                error: function (error) {
-                    console.error(error.responseText);
-                    bus.$emit('mediaListMoved', error.responseText);
+            confirmDialog({title: $("#moveMedia").data("title"), message: $("#deleteMedia").data("title"), callback: function (resp) {
```

Comment:
why use both #moveMedia or #deleteMedia here ?

Reply:
Good catch, may be it was copy paste issue from the my latest change